manifests/init.pp

```diff
@@ -64,14 +67,16 @@
   $package_name            = $nsclient::params::package_name,
   $download_destination    = $nsclient::params::download_destination,
   $config_template         = $nsclient::params::config_template,
-  $install_path            = $nsclient::params::install_path
+  $install_path            = $nsclient::params::install_path,
+  $password                = $nsclient::params::password
```

Comment:
Clean you please add a proper datatype to the new parameter? A quote from our [docs](https://voxpupuli.org/docs/#reviewing-a-module-pr):

> Are new parameters introduced? They must have datatypes

> Are datatypes from stdlib used? Ensure that lowest supported stdlib version is 4.13.1. Check if a newer version introduced the used datatype